### DISCUSSION TITLE
How do I know a package not support like windows system.

### DISCUSSION THREAD

**Discussion by bitnick10** [microsoft/vcpkg#19993] [Q&A] [answered]
C++ [boost](https://www.boost.org/) should be support linux windows and others.But some package only support windows or only support linux,How do I know which it does not support?

**Comment by Dobby233Liu** [comment, marked as answer]
Visit https://vcpkg.io/en/packages.html and search for the package you're looking for. The page for the package you're looking for should tell you what platforms are supported.
Or look at the vcpkg.json of the port in this repository (e.g. `ports/<port name>/vcpkg.json`), there is a "supports" property that will tell; If there is none, look at the portfile (e.g. `ports/<port name>/portfile.cmake`), some ports will intentionally fail when it's built for some specific platforms; If you found nothing, then try looking at the CONTROL file, though I don't know it's format.
Hope this helps
